.github/workflows/automated-tests.yml

```
@@ -369,6 +369,43 @@ jobs:
           npm ci
           npx playwright install-deps
           npx playwright install
+      - name: Prepare flaky test exclusions
+        working-directory: ./bigbluebutton-tests/playwright
+        run: |
+          # Read flaky tests from core repo and build a grep-invert parameter
+          FLAKY_TESTS_FILE="sdk-flaky-tests.txt"
+          FLAKY_TESTS_LINES=""
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)